string update_command
string action_to_be_modified
string action_to_be_added
string planName
---
string response
